core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -29,7 +29,7 @@ import org.apache.spark.annotation.DeveloperApi
 class TaskInfo(
     val taskId: Long,
     val index: Int,
-    val attempt: Int,
```

Comment:
Maybe we need to take a look at the code to see the meaning of `attempt`.  I find https://github.com/apache/spark/blob/branch-1.3/core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala#L451-L452. Looks like it is indeed `attemptNumber`.